### PR TITLE
ztp: Add an initial example of a DU profile

### PR DESCRIPTION
**Pull request by lack**
Interestingly enough, the main differences between SNO and multi-node
site configurations are in the 'group-du-ranGen' vs
'group-du-sno-ranGen' level.  The actual site-specifics look very
similar (apart from the MCP role name)

Signed-off-by: Jim Ramsay <jramsay@redhat.com>

### CODE REVIEW

**Comment by lack** [comment]
Tested - The resulting CRs look pretty reasonable and work well on a standard cluster.

File: ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-ranGen.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du"
```
Comment (serngawy) [inline ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-ranGen.yaml:5]: duplicated name as group-du-3node-ranGen.yaml .

**Review by serngawy — COMMENTED** [review]
policies sounds good to me, just make sure u can run policyGen kustomize plugin without raising errors  AND add those files to kustomization.yaml with comment or notes

File: ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-ranGen.yaml (hunk 1 above)
Comment (lack) [reply ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-ranGen.yaml:5]: Good catch!  I just found I had to introduce a differentiation between the 3-node and standard clusters.  I'll rename to match.

Comment (lack) [reply ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-ranGen.yaml]: Done.

**Comment by lack** [comment]
@serngawy I have updated the example kustomization.yaml and when I run the policygenerator kustomize on it, it does not error and creates all policies as expected.

**Review by serngawy — APPROVED** [review]

**Pull request merged**